src/index.ts

```diff
@@ -1,11 +1,6 @@
 /* eslint-disable no-console */
 
-import {
-	onGuConsentNotification,
-	onIabConsentNotification,
-	shouldShow,
-} from '@guardian/old-cmp';
-import { ConsentManagementPlatform } from '@guardian/old-cmp/dist/ConsentManagementPlatform';
+import { oldCmp as olderCmp } from './oldCmp';
```

Comment:
Is there a way to directly import the old CMP into an object, without having two `init` in the same scope?

Reply:
Yes, [you just need the curly braces around the exported object](https://javascript.info/import-export):

```js
import { oldCmp } from './oldCmp';
export { oldCmp };
```